model/host/host.go

```diff
@@ -2037,6 +2044,11 @@ func (h *Host) GetElapsedCommunicationTime() time.Duration {
 	return time.Since(h.CreationTime)
 }
 
+// GetElapsedTeardownTime calculates the elapsed time since the teardown of the task group started.
+func (h *Host) GetElapsedTeardownTime() time.Duration {
```

Comment:
since the only place we use this we check the value against a global variable, could we just rename this be like "TeardownTimeExceeded" or something, and move that check in here? (i.e. `time.Since(h.TaskGroupTeardownStartTime) > evergreen.MaxTeardownGroupThreshold`)